apps/src/templates/teacherHomepage/SectionsTable.jsx

```diff
@@ -0,0 +1,151 @@
+import React, { PropTypes } from 'react';
+import color from "@cdo/apps/util/color";
+import i18n from '@cdo/locale';
+
+const styles = {
+  table: {
```

Comment:
I don't know of any prior discussion, but that is a good point that at some point we should think about rationalizing this. This could be done either by having a set of styles that you import, or by having a base Table class that starts out with a set of styles.